### PR TITLE
Improved navbar functionality and adaptability across different scree…

### PR DESCRIPTION
**Pull request by Amitooo1**
Feat #68  

Feature enhancement:
Now Navbar is adaptable to all the screen sizes without any breaks or overflow. More shadow gives a better visual appearance and more and be seen in video below...

https://github.com/Mansi168/EmoWell/assets/108569153/5d76efe8-16e8-45ac-ab7a-dc578720d004

## Checklist:
- [ X] I have performed a self-review of my code
- [ X] I have read and followed the Contribution Guidelines.
- [ X] I have tested the changes thoroughly before submitting this pull request.
- [ X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by Amitooo1** [comment]
Hey @everly-gif, could you please review this PR?

**Comment by everly-gif** [comment]
Hi, looks good, was this design approved by the maintainer? If not I would suggest the width of the menu items in hamburger mode to span full width @Amitooo1

**Review by everly-gif — APPROVED** [review]

**Comment by Amitooo1** [comment]
@everly-gif if @Mansi168 will also wants me to change the menu bar width to full, I'll do that also, but personally this seems more appealing to me.
@Mansi168 please review this PR,

**Pull request merged**
